source/_integrations/waze_travel_time.markdown

```diff
@@ -62,6 +62,62 @@ template:
 ```
 
 {% endraw %}
+Automations / Scrips can be setup to poll Waze on any changes to the input_select.
+An example of a script:
+```yaml
+sequence:
```

Comment:
We do not need a script for this, it adds additional unneeded complexity.

Reply:
This script allows a user to get immediate updates. It does not require you to do this but if you want to get the time updated you click the button. I do not know why you think this makes it more complex?

Reply:
More references adds more complexity. And since it is not needed to use a script, it adds complexity.

Reply:
I agree that more references add more complexity. Not sure why adding an example that allows a user to setup an immediate update adds to this? If it is not wanted or needed that is fine.  I again, tried to add examples to the documentation on areas that took me a lot of time to figure out. I still believe the document should be mostly self contained or at the least have links to other documents that show a user how to do something. If you look at a similar application like Google travel there is examples included in the document.

Reply:
Please don't use a separate script in the example. Thanks.